units/Legion/T3/legjav.lua

```diff
@@ -0,0 +1,208 @@
+return {
+	legjav = {
+		buildpic = "legjav.DDS",
+		buildtime = 32000,
+		canmove = true,
+		cantbetransported = true,
+		collisionvolumeoffsets = "0 0 0",
+		collisionvolumescales = "56 72 56",
+		collisionvolumetype = "CYLY",
+		corpse = "DEAD",
+		energycost = 21000,
+		explodeas = "explosiont3med",
+		footprintx = 3,
+		footprintz = 3,
+		health = 7200,
+		idleautoheal = 16,
+		idletime = 1800,
+		mass = 1200,
+		maxacc = 0.1750,
+		maxdec = 0.7500,
+		maxslope = 17,
+		maxwaterdepth = 32,
+		metalcost = 1200,
+		movementclass = "HABOT4",
+		nochasecategory = "VTOL",
+		objectname = "Units/legjav.s3o",
+		script = "Units/legjav.cob",
+		seismicsignature = 0,
+		selfdestructas = "explosiont3",
+		sightdistance = 455,
+		speed = 62.0,
+		turninplace = true,
+		turninplaceanglelimit = 90,
+		turninplacespeedlimit = 1.98,
+		turnrate = 900,
+		upright = true,
+		customparams = {
+			maxrange = "350",
+			model_author = "Tharsis",
+			normaltex = "unittextures/leg_normal.dds",
+			subfolder = "Legion/T3",
+			techlevel = 3,
+			unitgroup = "weapon",
+			weapon1turretx = 90,
+			weapon1turrety = 150,
+		},
+		featuredefs = {
+			dead = {
+				blocking = true,
+				category = "corpses",
+				collisionvolumeoffsets = "0 0 0",
+				collisionvolumescales = "60 30 60",
+				collisionvolumetype = "Box",
+				damage = 2400,
+				featuredead = "HEAP",
+				footprintx = 3,
```

Comment:
pending https://github.com/beyond-all-reason/Beyond-All-Reason/pull/4083 please change this to what is agreed to for the regular model